packages/taquito-rpc/src/taquito-rpc.ts

```diff
@@ -458,7 +458,6 @@ export class RpcClient implements RpcClientInterface {
       'endorsing_reward_per_slot',
       'double_baking_punishment',
       'delay_increment_per_round',
-      'tx_rollup_commitment_bond'
```

Comment:
What is the reason for removing the property? it might be a mistake

Reply:
definitely unintentional 😮